src/content/guides/shimming.md

```diff
@@ -13,228 +13,359 @@ related:
 
-The simplest way to mitigate this is by adding a separate entry point in your webpack config file including the polyfills your project needs.
+    element.innerHTML = join(['Hello', 'webpack'], ' ');
 
-```javascript
-// webpack.config.js
-module.exports = {
-  entry: {
-    polyfills: [
-      'babel-polyfill',
-      'whatwg-fetch'
-    ],
-    main: './src/index.js'
+    return element;
   }
-  // ... rest of your webpack config
-};
+
+  document.body.appendChild(component());
+```
+
+__project__
+
+``` diff
+  webpack-demo
+  |- package.json
+  |- webpack.config.js
+  |- /dist
+  |- /src
+    |- index.js
+    |- globals.js
++   |- polyfills.js
++   |- fetch.js
+  |- /node_modules
 ```
 
-An alternative is to create a new entry file and manually import these packages.
+__src/polyfills.js__
 
 ```javascript
-// src/polyfills.js
 import 'babel-polyfill';
 import 'whatwg-fetch';
 ```
 
-```javascript
-// webpack.config.js
-module.exports = {
-  entry: {
-    polyfills: './src/polyfills.js',
-    main: './src/index.js'
-  }
-  // rest of your webpack config
-};
-```
+With that in place, we can add the logic to conditionally load `src/polyfills.js` depending on whether or not. How you make this decision depends on the technologies and browsers you need to support. We'll just do some simple testing before we run our fetching script:
 
-In your html file you need to conditionally load the `polyfills.js` file before your bundle. How you make this decision depends on the technologies and browsers you need to support.
+__src/index.js__
 
-```html
-<script>
-  var modernBrowser = (
-    'fetch' in window &&
-    'assign' in Object
-  );
+``` diff
+  function component() {
+    var element = document.createElement('div');
 
-  var scripts = [ '/main.js' ];
+    element.innerHTML = join(['Hello', 'webpack'], ' ');
 
-  if (!modernBrowser) {
-    scripts.unshift('/polyfills.js');
+    return element;
   }
 
-  scripts.map(function(src) {
-    var scriptElement = document.createElement('script');
-    scriptElement.async = false;
-    scriptElement.src = src;
-    document.head.appendChild(scriptElement);
-  });
-</script>
+  document.body.appendChild(component());
++
++ let modernBrowser = (
++   'fetch' in window &&
++   'assign' in Object
++ );
++
++ if ( !modernBrowser ) {
++   import('./polyfills.js', /* webpackChunkName: "polyfills" */).then(
++     import('./fetch.js' /* webpackChunkName: "fetch" */)
++   )
++
++ } else import('./fetch.js' /* webpackChunkName: "fetch" */)
 ```
 
-T> Any script added dynamically like in the example above will run as soon as it's parsed, but we need our polyfill to run before our bundle. This is why we are setting `async` to `false` for each script.
+__src/fetch.js__
 
+``` js
+fetch('https://jsonplaceholder.typicode.com/users')
+  .then(response => response.json())
+  .then(json => {
+    console.log('We retrieved some data! AND we're confident it will work on a variety of browser distributions.')
```

Comment:
`we\'re`

Reply:
Good catch, thanks! Will fix that now as I'm wrapping up the remaining content. What'd you think of the rest of the updates?

Reply:
I'm not sure about the `fetch`/`polyfills` example: I probably wouldn't use it like that, because relying on *global* side effects when doing *dynamic* imports sounds wrong and just begs for race conditions. The previous example where they used `script` tags is more realistic.

Also, why did you remove `noParse` docs - are they mentioned elsewhere?

Reply:
Just added `noParse` back to the bottom in _Other Utilities_. As for the `fetch` / `polyfills` example, yeah I think you may be right. I mentioned why I did it in my comment below but maybe I'll change that back. Thanks for your feedback! Feel free to do another review now that everything is pushed.